tests/testdemographics.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * @fileoverview Defines tests for audience demographics.
+ */
+
+
+/** @const {!RegExp} */ var PATTERN = /^\d([,\.]*\d)*%$/;
+
+/** @const {!Object.<string, function(NodeList):boolean>} */ var VALIDATORS = {
+  'gender-data-table': function(elements) {
+    return testStaticDataTable(elements, ['male', 'female']);
+  },
+  'browsing-location-data-table': function(elements) {
+    return testStaticDataTable(elements, ['home', 'school', 'work']);
+  },
+  'geo-data-table': function(elements) {
+    /** @type {number} */ var length = elements.length;
+    /** @type {number} */ var index = 0;
+    /** @type {string} */ var content;
+    /** @type {NodeList} */ var cells;
+    /** @type {Node} */ var table = document.getElementById('geo-data-table');
+    /** @type {Node} */ var parentNode = table && table.parentNode;
```

Comment:
Please put variables in following order:
- variables with values;
- variables w/o values;